### DISCUSSION TITLE
Software available on CoCalc

### DISCUSSION THREAD

**Discussion by haraldschilly** [sagemathinc/cocalc#6167] [Show and tell]
## General
- The pages on cocalc about available software are here: https://cocalc.com/software

##  Documentation

- Python 3
  - [Pandas](https://pandas.pydata.org/docs/) high-performance, easy-to-use data structures and data analysis tools

- Sage
  - [Sagemath](https://doc.sagemath.org/html/en/) open-source mathematics software

**Comment by williamstein** [comment]
The pages on cocalc about available software are here: https://cocalc.com/software
